pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -302,6 +302,7 @@ func (c *Controller) syncOwnedProvisionRequest(ctx context.Context, wl *kueue.Wo
 			}
 
 			if err := c.client.Create(ctx, req); err != nil {
+				c.record.Eventf(wl, corev1.EventTypeWarning, "ProvisioningRequestError", "Error creating ProvisioningRequest: %q", req.Name)
```

Comment:
It is a bit weird seeing that you title the constant as ProvisioningRequestError and the type of event is a warning.

I would also declare this as a constant and move it to the constants package.